examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scroll_view/custom_scroll_view.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('What should be visible or not visible in the initial state.', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('What should be visible in the initial state.', (WidgetTester tester) async {
```
Try to keep it simple, it's not easy to find good test description.